Makefile

```diff
@@ -84,6 +83,10 @@ fuzz: bin/cmd-fuzz-query.zip
 fuzzv: bin/cmd-fuzz-query.zip
 	FUZZDEBUG=1 go-fuzz -bin=./bin/cmd-fuzz-query.zip -workdir=workdir/query -testoutput -procs 1
 
+fuzzcoverage:
```

Comment:
fails on go 1.6 for me (with test.Run in fuzz_test.go)

Reply:
I'll ad a build tag.